prime-router/settings/staging/0170-ucsd-onboarding-receiver-settings.yml

```diff
@@ -0,0 +1,101 @@
+# To submit this to staging, run:
+# ./prime multiple-settings set --env staging --input ./settings/staging/0170-ucsd-onboarding-receiver-settings.yml
+#
+# To submit this in localhost, run:
+# ./prime multiple-settings set --input ./settings/staging/0170-ucsd-onboarding-receiver-settings.yml
+#
+# Description:
+#   These following settings are for UCSD
+#   Result Receiver (ucsd.etor-nbs-results)
+#
+---
+- name: "ucsd"
+  description: "University of California San Diego"
+  jurisdiction: "STATE"
+  stateCode: CA
+  countyName: null
+  filters: []
+  featureFlags: []
+  keys: null
+  senders: []
+  receivers:
+    - name: "etor-nbs-results"
+      organizationName: "ucsd"
+      topic: "etor-ti"
+      customerStatus: "active"
+      translation: !<HL7>
+        schemaName: "metadata/hl7_mapping/ORU_R01/ORU_R01-base"
```

Comment:
The schema should be: `classpath:/metadata/hl7_mapping/ORU_R01/ORU_R01-base.yml`

Reply:
Good catch. Yes, with the previous changes, we need to add `classpath:`